sys/net/application_layer/nanocoap/cache.c

```diff
@@ -202,15 +211,15 @@ int nanocoap_cache_process(const uint8_t *cache_key, unsigned request_method,
         if (ce) {
             /* set max_age to now(), so that the cache is considered
              * stale immdiately */
-            ce->max_age = xtimer_now_usec64();
+            ce->max_age = ztimer_now(ZTIMER_SEC);
         }
     }
     /* This response is cacheable: Caches can use the Max-Age Option
        to determine freshness and (if present) the
        ETag Option for validation.
     */
     else if (resp->hdr->code == COAP_CODE_CONTENT) {
-        uint64_t now = xtimer_now_usec64();
+        uint32_t now = ztimer_now(ZTIMER_SEC);
         /* cache entry is stale */
         if (ce && (ce->max_age < now)) {
```

Comment:
this maybe won't be "considered stale immediately" with ZTIMER_SEC?

Reply:
wrt to https://github.com/RIOT-OS/RIOT/issues/17808: this line was removed in https://github.com/RIOT-OS/RIOT/pull/17801/commits/6b3a2729f9910f3450daa87c6c1ca0170e29b6b8 (which is soon to be cherry-picked into https://github.com/RIOT-OS/RIOT/pull/13889), and was thus not passed in the setup described in https://github.com/RIOT-OS/RIOT/issues/17808.